src/chrome/content/rules/Samsung.com.xml

```diff
@@ -0,0 +1,157 @@
+		<test url="http://opensource.samsung.com/js" />
+		<test url="http://opensource.samsung.com/js2" />
+		<test url="http://opensource.samsung.com/reception/receptionSub" />
+		<test url="http://opensource.samsung.com/reception/receptionSub.do" />
+		<!--	To http	-->
+		<test url="http://opensource.samsung.com/" />
+		<test url="http://opensource.samsung.com/images" />
+		<!--test url="http://opensource.samsung.com/community.do/" /-->
+		<test url="http://opensource.samsung.com/org.ditchnet.taglib" />
+		<test url="http://opensource.samsung.com/popup" />
+		<test url="http://opensource.samsung.com/reception/" />
+		<test url="http://opensource.samsung.com/reception.do/" />
+		<test url="http://opensource.samsung.com/scmShowFileRevision" />
+		<test url="http://opensource.samsung.com/stylesheet" />
+		<test url="http://opensource.samsung.com/usageStatistics" />
+		<test url="http://opensource.samsung.com/wikiTreeMessage" />
+
+	<rule from="^http://opensource\.samsung\.com/" to="https://opensource.samsung.com/" />
+		
+		<!-- Support page FAQ and other features are broken. Also, their i18n is a lie.
+		     Github issue: https://github.com/EFForg/https-everywhere/issues/309 -->
+		<exclusion pattern="^http://www\.samsung\.com/us/(?:support/|function/)" />
+			<test url="http://www.samsung.com/us/function/" />
+			<test url="http://www.samsung.com/us/support/" />
+
+	<!--	Not secured by server:
+					-->
+	<!--securecookie host="^\.samsung\.com$" name="^(_common_country|_common_lang|COUNTRY_CODE|EXPAND_SCHEDULED|REMEMBERED_DEVICE_CODE_CAN)$" /-->
+	<!--securecookie host="^apps\.samsung\.com$" name="^WMONID$" /-->
+
+	<securecookie host="^(?!shop\.us\.).*\.samsung\.com$" name=".+" />
+
+	<target host="*.samsung.com" />
+
+	<rule from="^http://www\.samsung\.com/"
+		to="https://www.samsung.com/" />
+		<test url="http://www.samsung.com/cn/common/css/home.css" />
+		
+	<rule from="^http://nmetrics\.samsung\.com/"
+		to="https://samsung-com.112.2o7.net/" />
+		<test url="http://nmetrics.samsung.com/b/ss/sssamsung4in" />
+		
+	<rule from="^http://((?:stg-)?account|cdn|(?:help\.|img\.|pub\.)?content|findmymobile|images|link|secureus|smetrics|support-cn|us|shop\.us)\.samsung\.com/"
+		to="https://$1.samsung.com/" />
+		<test url="http://account.samsung.com/membership/service/getServieInfoList.do/" />
+		<test url="http://cdn.samsung.com/common/next/css/apps/mobile/barshade.css" />
+		<test url="http://help.content.samsung.com/csweb/main/main.do" />
+		<test url="http://findmymobile.samsung.com/img/favicon02.ico/" />
+		<test url="http://images.samsung.com/" />
+		<test url="http://link.samsung.com/" />
+		<test url="http://secureus.samsung.com/us/mobile/galaxy-tab/SM-T810NZWEXAR" />
+		<test url="http://smetrics.samsung.com/" />
+		<test url="http://support-cn.samsung.com/cn/common/css/local_global_common.css" />
+		<test url="http://us.samsung.com/us/index.jsp" />
+		<test url="http://shop.us.samsung.com/store?Action=DisplayDrcartSummary" />
```

Comment:
Please also add test urls for:
stg-account.samsung.com
img.content.samsung.com
pub.content.samsung.com